lib/cash-in/cash-in-tx.js

```diff
@@ -8,6 +8,7 @@ const plugins = require('../plugins')
 const logger = require('../logger')
 const settingsLoader = require('../new-settings-loader')
 const configManager = require('../new-config-manager')
+const notifier = require("../notifier/index")
```

Comment:
Same applies here and you're using double quotes on the import, instead of single quotes.

```suggestion
const notifier = require('../notifier')
```